src/pages/profile-page/personal-profile-page/personal-profile-page.tsx

```diff
@@ -42,6 +45,7 @@ const AsyncPersonalProfilePage = () => {
         <PageHeader label="my profile" />
       </div>
       <UserDetails
+        profilePicUrl={profilePicUrl}
         username={userData?.username ?? ''}
         email={userData?.email}
         dateJoined={new Date()}
```

Comment:
do we have this data?

Reply:
It wasn't in the interface, so I wasn't sure if it existed yet. It's there; and I've fixed it